### ISSUE TITLE
i made a bunch of edits, up through and including 2.1

### ISSUE THREAD

**Issue by jovo** [neurodata/MGC-paper#52]
this included deleting some of your text, and adding a bunch.

note that i am making it _way_ too verbose for now.
we will filter back down eventually.
first, i want to make sure we get all the concepts in clearly.
then, we can revise accordingly.

to delete stuff, i just added a note \cs{.}.

this issue is: for everything that i deleted, if you agree, just delete the note.
there is no rush on this, we can always ignore all the edits by changing the package 'changes' option.
in other words, i'd rather you finish the figs first.

**Comment by cshen6** [comment]
ok. I just synchronized my in-progress solved tex issues with your tex change.

Most figure issues are solved, just a few more...

**Comment by cshen6** [comment]
there is only one main thing I disagree on, which is section 2.1 the definition of general corr and the follow-on local corr definition. 

I have battled this for a while in my mind last month; and we should talk on this one.

**Comment by jovo** [comment]
ok, i did what friedman 83 did, which was much simpler than what we had
previous.  we can discuss sometime today (thurs) via skype if you like?

On Thu, Jun 9, 2016 at 4:39 AM, cshen6 notifications@github.com wrote:

> there is only one main thing I disagree on, which is section 2.1 the
> definition of general corr and the follow-on local corr definition.
> 
> I have battled this for a while in my mind last month; and we should talk
> on this one.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/jovo/RankdCorr/issues/52#issuecomment-224786590, or mute
> the thread
> https://github.com/notifications/unsubscribe/AACjcq9lWzK430GoRzInrfrt1UEwp__Jks5qJ3zSgaJpZM4IxElK
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

**Comment by cshen6** [comment]
I went through the tex and deleted all \cs comments, except \cs{cite CPAC} and \cs{...} in discussion.

A few things:
1. In definition of general corr in section 2.1, z should be the product of standard deviations.
Then I added back the sample means, because they are really necessary in the definition. 
I think it won't complex stuff, because it is just mean (and we already have std :-)
Then in appendix before global dcorr, I specified what a and b are for dcorr/mcorr/mantel.
1. note that we always use MGC-mcorr, for outlier / real data. 
   MGC-dcorr and MGC-mantel only appears in 1d and hd simulations.
2. for the outlier model "The robustness still holds as the dimension, sample size, or the underlying dependency change (not shown)", which is true as I tried various simulations in our setting.
3. I commented out the performance profiles section in appendix; we can comment back if you still want to show it. 
   Then I add the power threshold explanation to the caption in the power heatmap.
4. I removed figure letterhead in figure 4&6. Not sure if I should remove figure 7 letter head.

The only thing remaining for now, is to update figure 1&4, which I may ask your help later.

**Comment by cshen6** [comment]
@jovo 
Just updated everything on github, including figure 1,2,3; section 2.1 centering definition; appendix MGC section, as well as adjusting appendix proof based on your theorem write-ups.

I should have updated yesterday; but I was weighing and investigating the pros and cons of centering in section 2.1. In any case, it is now in accordance with what you wanted.

Figure 1 is arranged like what we discussed three days ago, with some small tweaks for better visualization. But I am pretty sure you would ask for some cosmetic changes there :-)

**Comment by jovo** [comment]
when i compile, fig1 looks like the new one,
but the other figures look like the older versions.
i love eq's 1--3!

On Wed, Jun 15, 2016 at 2:16 PM, cshen6 notifications@github.com wrote:

> @jovo https://github.com/jovo
> Just updated everything on github, including figure 1,2,3; section 2.1
> centering definition; appendix MGC section, as well as adjusting appendix
> proof based on your theorem write-ups.
> 
> I should have updated yesterday; but I was weighing and investigating the
> pros and cons of centering in section 2.1. Let me know what you think now
> about the definition.
> 
> Figure 1 is arranged as what we discussed previously, with some small
> tweaks for better visualization. But I am pretty sure you would ask for
> some cosmetic changes there :-)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/jovo/RankdCorr/issues/52#issuecomment-226273885, or mute
> the thread
> https://github.com/notifications/unsubscribe/AACjcj99AFuozR4LFrGvMGu35C16QBt9ks5qMEGagaJpZM4IxElK
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

**Comment by cshen6** [comment]
Great!

For figure 2 &3, I thought we only include MGC_{mcorr}, and all benchmarks?
Or you mean we only include MGC_{mcorr}, Mcorr, HHG in figure 2 & 3? (And
others in appendix)

On Wed, Jun 15, 2016 at 3:03 PM, joshua vogelstein <notifications@github.com

> wrote:
> 
> when i compile, fig1 looks like the new one,
> but the other figures look like the older versions.
> i love eq's 1--3!
>

**Comment by jovo** [comment]
fig 2 & 3 only contain mgc, mcorr, and hhg.

fig a1 is wrong again.

others are ok.

On Wed, Jun 15, 2016 at 3:23 PM, cshen6 notifications@github.com wrote:

> Great!
> 
> For figure 2 &3, I thought we only include MGC_{mcorr}, and all benchmarks?
> Or you mean we only include MGC_{mcorr}, Mcorr, HHG in figure 2 & 3? (And
> others in appendix)
> 
> On Wed, Jun 15, 2016 at 3:03 PM, joshua vogelstein <
> notifications@github.com
> 
> > wrote:
> > 
> > when i compile, fig1 looks like the new one,
> > but the other figures look like the older versions.
> > i love eq's 1--3!
> > 
> > On Wed, Jun 15, 2016 at 2:16 PM, cshen6 notifications@github.com
> > wrote:
> > 
> > > @jovo https://github.com/jovo
> > > Just updated everything on github, including figure 1,2,3; section 2.1
> > > centering definition; appendix MGC section, as well as adjusting
> > > appendix
> > > proof based on your theorem write-ups.
> > > 
> > > I should have updated yesterday; but I was weighing and investigating
> > > the
> > > pros and cons of centering in section 2.1. Let me know what you think
> > > now
> > > about the definition.
> > > 
> > > Figure 1 is arranged as what we discussed previously, with some small
> > > tweaks for better visualization. But I am pretty sure you would ask for
> > > some cosmetic changes there :-)
> > > 
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > https://github.com/jovo/RankdCorr/issues/52#issuecomment-226273885,
> > > or mute
> > > the thread
> > > <
> > 
> > https://github.com/notifications/unsubscribe/AACjcj99AFuozR4LFrGvMGu35C16QBt9ks5qMEGagaJpZM4IxElK
> > 
> > > .
> > 
> > ## 
> > 
> > the glass is all full: half water, half air.
> > neurodata.io, jovo calendar
> > <
> > 
> > https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York
> > 
> > 
> > —
> > You are receiving this because you were assigned.
> > Reply to this email directly, view it on GitHub
> > https://github.com/jovo/RankdCorr/issues/52#issuecomment-226287613,
> > or mute
> > the thread
> > <
> > https://github.com/notifications/unsubscribe/ALX0y8EJm_9SHTWXjxlrjL6Jur6fu8-Eks5qMEyGgaJpZM4IxElK
> > 
> > .
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/jovo/RankdCorr/issues/52#issuecomment-226292884, or mute
> the thread
> https://github.com/notifications/unsubscribe/AACjcraj0aTyGX8BgEmPpbm3WpE2ULj4ks5qMFE7gaJpZM4IxElK
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

**Comment by jovo** [comment]
but that's not what i get when i compile.

On Wed, Jun 15, 2016 at 3:27 PM, joshua vogelstein jovo@jhu.edu wrote:

> fig 2 & 3 only contain mgc, mcorr, and hhg.
> 
> fig a1 is wrong again.
> 
> others are ok.
> 
> On Wed, Jun 15, 2016 at 3:23 PM, cshen6 notifications@github.com wrote:
> 
> > Great!
> > 
> > For figure 2 &3, I thought we only include MGC_{mcorr}, and all
> > benchmarks?
> > Or you mean we only include MGC_{mcorr}, Mcorr, HHG in figure 2 & 3? (And
> > others in appendix)
> > 
> > On Wed, Jun 15, 2016 at 3:03 PM, joshua vogelstein <
> > notifications@github.com
> > 
> > > wrote:
> > > 
> > > when i compile, fig1 looks like the new one,
> > > but the other figures look like the older versions.
> > > i love eq's 1--3!
> > > 
> > > On Wed, Jun 15, 2016 at 2:16 PM, cshen6 notifications@github.com
> > > wrote:
> > > 
> > > > @jovo https://github.com/jovo
> > > > Just updated everything on github, including figure 1,2,3; section 2.1
> > > > centering definition; appendix MGC section, as well as adjusting
> > > > appendix
> > > > proof based on your theorem write-ups.
> > > > 
> > > > I should have updated yesterday; but I was weighing and investigating
> > > > the
> > > > pros and cons of centering in section 2.1. Let me know what you think
> > > > now
> > > > about the definition.
> > > > 
> > > > Figure 1 is arranged as what we discussed previously, with some small
> > > > tweaks for better visualization. But I am pretty sure you would ask
> > > > for
> > > > some cosmetic changes there :-)
> > > > 
> > > > —
> > > > You are receiving this because you were mentioned.
> > > > Reply to this email directly, view it on GitHub
> > > > https://github.com/jovo/RankdCorr/issues/52#issuecomment-226273885,
> > > > or mute
> > > > the thread
> > > > <
> > > 
> > > https://github.com/notifications/unsubscribe/AACjcj99AFuozR4LFrGvMGu35C16QBt9ks5qMEGagaJpZM4IxElK
> > > 
> > > > .
> > > 
> > > ## 
> > > 
> > > the glass is all full: half water, half air.
> > > neurodata.io, jovo calendar
> > > <
> > > 
> > > https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York
> > > 
> > > 
> > > —
> > > You are receiving this because you were assigned.
> > > Reply to this email directly, view it on GitHub
> > > https://github.com/jovo/RankdCorr/issues/52#issuecomment-226287613,
> > > or mute
> > > the thread
> > > <
> > > https://github.com/notifications/unsubscribe/ALX0y8EJm_9SHTWXjxlrjL6Jur6fu8-Eks5qMEyGgaJpZM4IxElK
> > > 
> > > .
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > https://github.com/jovo/RankdCorr/issues/52#issuecomment-226292884, or mute
> > the thread
> > https://github.com/notifications/unsubscribe/AACjcraj0aTyGX8BgEmPpbm3WpE2ULj4ks5qMFE7gaJpZM4IxElK
> > .
> 
> ## 
> 
> the glass is all full:  half water, half air.
> neurodata.io, jovo calendar
> https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York
